### ISSUE TITLE
rtl support

### ISSUE THREAD

**Issue by zozolapino** [albertogasparin/Optiscroll#25]
Hi!

Using the 3.0 version I tried to set .optiscroll-v {left:0;} and .optiscroll-vtrack {...left:6px;} with direction:rtl:
While the original scrollbar disappears, the drag zone is still under the original scrollbar and the new bar is 6px at right.

Thanks for your time!

**Comment by zozolapino** [comment]
I forget to say I changed   hideNativeScrollbars: function this way:

```
...
    var size = G.nativeScrollbarSize,
        scrollElStyle = scrollEl.style;
    if(size === 0) {
      // hide Webkit/touch scrollbars
      var time = Date.now();
      scrollEl.setAttribute('data-scroll', time);
      return Utils.addCssRule('[data-scroll="' + time + '"]::-webkit-scrollbar', 'display:none;width:0;height:0;');
    } else {
if( GS.rtl == true ) {
	scrollElStyle.left = -size + 'px';
...
```

**Comment by albertogasparin** [comment]
Rtl support is actually a good idea. 
Do you think is it better as an option (like you did) or by checking the directionality of the element (with `getComputedStyle(el).direction === 'rtl'`)?

Anyway, I've tried to set `html[dir="rtl"]` to Optiscroll website and as you wrote, I had to change the js a bit and the style to be:
``` css
.optiscroll-v { left:0; }
.optiscroll-vtrack { left: 4px; } 
.optiscroll.has-vtrack.has-htrack > .optiscroll-h { left: 8px; right: 4px; }
```
However, what do you mean by "the drag zone is still under the original scrollbar"? Are you referring to the horizontal scrollbar?

**Comment by zozolapino** [comment]
Checked again but still the same problem with windows 10/chrome: the (trackbar) drag and drop zone is still where the old scrollbar was and the vertical grayed trackbar is displayed 6/10 pixels at right of this drag zone. Like a ghost effect. Strangely there is no problem for the div I use as the main wrapper (width/height:100%). No problem too with firefox/edge/ie11.  However with these browsers the text is hilighted when I move the trackbar. I don't use horizontal scrollbars.

getComputedStyle(el).direction === 'rtl' in your code would be nice!

**Comment by albertogasparin** [comment]
Please have a look at `v3` branch. Run `grunt build` and open the `index.html` file. The scrollable box on the right is `rtl` and it should work ok.

**Comment by zozolapino** [comment]
Thank-you Alberto!

It works fine on chrome but still buggy on FF/Edge (divs text highlighted when moving the trackbars). I migrated to NiceScroll which works fine at the price of 60KB minified + JQuery... :/

**Comment by albertogasparin** [comment]
Well, I've re-checked the code and tested it on FF/Edge: when dragging the scrollbars the selection is prevented. 
However the default style of the scrollbars make dragging them quite tricky, so I've noticed that if you miss the target then you are going to trigger the selection. But that is an UX issue.
Anyway, FYI v3 will be ready soon.

**Comment by albertogasparin** [comment]
Merged #26
